### PR TITLE
ref(ratelimit): Added a config and changed a default

### PR DESCRIPTION
**Pull request by AniketDas-Tekky**
Org rate limits should be the most lenient rate limit. This change increases the default org limits to match the current highest, users.
I also made enforce_rate_limit a configurable option so that self-hosted can disable it from config.py

### CODE REVIEW

File: src/sentry/conf/server.py [hunk 1]
```diff
@@ -1329,11 +1329,12 @@ def create_partitioned_queues(name):
 
 # Rate limiting backend
 SENTRY_RATELIMITER = "sentry.ratelimits.base.RateLimiter"
+SENTRY_RATELIMITER_ENABLED = False
 SENTRY_RATELIMITER_OPTIONS = {}
 # These values were determined from analysis on one week of api access logs
 SENTRY_RATELIMITER_DEFAULT_IP = 1000
 SENTRY_RATELIMITER_DEFAULT_USER = 3230
-SENTRY_RATELIMITER_DEFAULT_ORG = 620
+SENTRY_RATELIMITER_DEFAULT_ORG = 3230
```
Comment (vladkluev) [inline src/sentry/conf/server.py:1337]: can we make user `620`? Also, is this overridable without deploy?

File: src/sentry/api/base.py [hunk 2]
```diff
@@ -103,7 +103,7 @@ class Endpoint(APIView):
     # Default Rate Limit Values, override in subclass
     # Should be of format: { <http function>: { <category>: RateLimit(limit, window) } }
     rate_limits: Mapping[str, Mapping[RateLimitCategory | str, RateLimit]] = {}
-    enforce_rate_limit: bool = False
+    enforce_rate_limit: bool = settings.SENTRY_RATELIMITER_ENABLED
```
Comment (vladkluev) [inline src/sentry/api/base.py:106]: can you override this without a deploy?

Comment (AniketDas-Tekky) [reply src/sentry/api/base.py:106]: Not currently.

File: src/sentry/conf/server.py (hunk 1 above)
Comment (AniketDas-Tekky) [reply src/sentry/conf/server.py]: Dropped them down. The settings aren't overridable without deploy at the moment. We probably need a new system to allow that.

**Review by volokluev — APPROVED** [review]

**Pull request merged**